CMake/External_OpenCV.cmake

```diff
@@ -200,6 +236,11 @@ if (CMAKE_COMPILER_IS_GNUCC AND NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 6)
   list(APPEND OpenCV_EXTRA_BUILD_FLAGS -DENABLE_PRECOMPILED_HEADERS:BOOL=OFF)
 endif()
 
+# OpenCV 3.3.0 has an option to enable C++ 11
+if (fletch_BUILD_CXX11)
+  list(APPEND OpenCV_EXTRA_BUILD_FLAGS -DENABLE_CXX11:BOOL=ON)
```

Comment:
I would keep it. I would like consistency across fletch for cxx 11/98, as much as possible. All that said, I feel like we should only support cxx11 compilers anyway, and likely make this an always on thing across fletch considering it's main purpose is to support kwiver. That is a larger discussion though and not for this PR.